src/ARMS/chassis.cpp

```diff
@@ -325,6 +325,12 @@ void velocity(int t, int max) {
 	delay(t);
 }
 
+void velocity(int t, int left_max, int right_max) {
+	motorMove(leftMotors, left_max, true);
+	motorMove(rightMotors, right_max, true);
```

Comment:
We probably want to make the function _either_ a single velocity or separate velocities for each side.
```suggestion
	motorMove(right_speed == 0 ? left_max : right_max, true);
```
Doing this would require adding a default value for the right speed in chassis.h as well.